src/api/app/models/staging/staged_requests.rb

```diff
@@ -84,6 +85,34 @@ def requests
     staging_workflow.unassigned_requests.where(number: request_numbers)
   end
 
+  def link_package(bs_request_action)
+    request = bs_request_action.bs_request
+
+    source_package = Package.get_by_project_and_name!(bs_request_action.source_project,
+                                                      bs_request_action.source_package)
+
+    # get the base revision from the source package
+    baserev = source_package.backend_package.srcmd5
+
+    # create the package in the staging project
+    link_package = Package.create!(project: staging_project,
+                                   name: bs_request_action.source_package)
```

Comment:
Does `create!` fail if it already exists? Because that can happen on supersedes

Reply:
hm supersedes isn't an issue, if you unstage the old issue and stage the new one? Both steps could be packed together in the `osc ps` command. In another hand would be ok to keep the package if it exists and just update the `_link` file?

Reply:
IMO a supersede is a single action (especially as long as you don't stop the scheduler from looking at the project after that unstage) - and beging able to update the _link file is main reason why branch is so harmful

Reply:
> would be ok to keep the package if it exists and just update the _link file?

Yes, that is the idea.